docs/_v4_x/1_setup.md

```diff
@@ -17,3 +17,44 @@ The client library is available on [Maven Central](https://search.maven.org/#sea
 #### Javadoc
 
 Javadoc is available here: [javadoc.io/doc/no.digipost.signature/signature-api-client-java](https://www.javadoc.io/doc/no.digipost.signature/signature-api-client-java/4.0)
+
+### Create client configuration
+A client configuration includes all organization specific configuration and all settings needed to connect to the correct environment for Posten signering.
+
+
+#### Load organization certificate
+
+The first step is to load the organization certificate (virksomhetssertifikat) through the `KeyStoreConfig`. It can be created from a Java Key Store (JKS) or directly from a PKCS12-container, which is the usual format of an organization certificate. The latter is the recommended way of loading it if you have the certificate stored as a simple file:
+
+
+``` java
+InputStream certificateStream = new FileInputStream("/Path/To/Certificate.p12");
```

Comment:
Lurer på om det er greit å ha røddig ressurs-håndtering i kodeeksemplene, samt benytte nyere APIer i JDKen:
```suggestion
KeyStoreConfig keyStoreConfig;
try (InputStream certificateStream = 
           Files.newInputStream(Paths.get("/Path/To/Certificate.p12"))) {
    keyStoreConfig = KeyStoreConfig.fromOrganizationCertificate(
           certificateStream, "passwordToCertificate");
}
```
Blir jo litt boilerplate, men burde ikke legge opp til at folk ikke slipper filressurser heller. F.eks. er eksemplet [i README.md](https://github.com/digipost/signature-api-client-java/blob/master/README.md#bygge-og-kj%C3%B8re-tester) forsiden ganske bad, da den åpner opp en filstream i initialiseringen av et statisk felt, men lukker aldri filstream'en.

Reply:
Litt fancyness der med suggested changes, men siden det ikke er endret over hele fjøla så kommer det helt nye commiter for dette.